modules/administration/pages/ssl-certs.adoc

```diff
@@ -0,0 +1,25 @@
+[[ssl-certs]]
+= SSL Certificates
+
+{productname} uses SSL certificates to ensure that clients are registered to the correct server.
+
+When you create an SSL certificate for your server, the certificate includes the host name of the server it is associated with.
+The certificate is then deployed on the server, and when clients attempt to connect, they check the certificate details.
+If the certificate contains the name of the server they are attempting to connect to, the connection will be successful.
+The client will also check if the certificate has been signed by a trusted certificate authority.
+
+All certificates must be signed by a certificate authority (CA), in order for them to be considered valid, and for clients to be able to successfully match against them.
+When an organization signs its own certificate, the certificate is considered self-signed.
+A self-signed certificate is easy to set up, and does not cost any money, but they are considered less secure.
```

Comment:
"easy" is one of the forbidden words ;)  [...] straight forward to set up?

Reply:
yep, straight-forward works